FEATURES.md

```diff
@@ -56,3 +56,8 @@ for a detailed explanation.
 
   - `interaction.<event-name>` for events handled by a component.
   - `interaction.ember-action` for closure actions.
+
+* `ember-runtime-enumerable-contains`
+
+Deprecates `Enumerable.contains` and `Array.contains` in favor of `Enumerable.includes` and `Array.includes` 
```

Comment:
I think `Enumerable#contains` is the convention in Ember even though it was originally a rubyism - at least it has the advantage that it's not valid syntax for something that is completely different